server/services/zwave/lib/comClass/defaultCommandClass.js

```diff
@@ -0,0 +1,83 @@
+const { transformValue, normalizeValue } = require('../utils/valueBinders');
+
+const DefaultCommandClass = function DefaultCommandClass() {};
+
+DefaultCommandClass.prototype = {
```

Comment:
Seems it's something remaining from the previous implementation

Reply:
What's the matter here? 🧐

Are you expecting something like this here:
```
function func1() {}
function func2() {}
function func3() {}

export default {
    func1,
    func2,
    func3
}
```
?

Reply:
I removed the prototype part... let me know if the matter is somewhere else.

Regards,

Reply:
Yes! You are just coding little helper functions, no need to create classes for that (those are stateless). The simpler, the better.